File: Java/BOJ/11536 줄 세우기.java
import java.io.*;
import java.util.*;
import java.math.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int N = Integer.parseInt(br.readLine());
        String[] name = new String[N];
        for (int i = 0 ; i < N; i++) {
        	name[i] = br.readLine();
        }
        String[] asname = Arrays.copyOf(name, N);
        String[] dename = Arrays.copyOf(name, N);
        Arrays.sort(asname);
        Arrays.sort(dename, (a, b) -> b.compareTo(a));
        
        if (Arrays.equals(name, dename)) {
            System.out.println("DECREASING");
        } else if (Arrays.equals(name, asname)) {
            System.out.println("INCREASING");
        } else {
            System.out.println("NEITHER");
        }
    }
}

File: Python/BOJ/11536 줄 세우기.py
import sys
N = int(sys.stdin.readline())
name = [sys.stdin.readline().strip() for _ in range(N)]
asname = sorted(name)
dename = sorted(name,reverse=True)
if name == dename :
    print('DECREASING')
elif name == asname :
    print('INCREASING')
else:
    print('NEITHER')